benchmarks/Tracing/ActivityCreationScenarios.cs

```diff
@@ -28,6 +28,13 @@ public static Activity CreateActivity(ActivitySource source)
             return activity;
         }
 
+        public static Activity CreateActivityWithKind(ActivitySource source)
+        {
+            var activity = source.StartActivity("name");
+            activity?.SetKind(ActivityKind.Client);
```

Comment:
pass activitykind as argument to .StartActivity itself. The setter is only for legacy activities we have to deal with in instrumentation projects.